HISTORY.rst

```diff
@@ -3,13 +3,24 @@
 Release History
 ---------------
 
-2.0.0 (2023-04-15) Wellton
+2.1.0 (2023-07-22) Youngtown
++++++++++++++++++++++++++++++
+
+This release creates `mesa.experimental` namespace this solves issue that PyPI release will not allow git-based install.
```

Comment:
Needs to say that since this is released right after, user needs to read 2.0.0 release note.